packages/annotations/package.json

```diff
@@ -27,7 +27,6 @@
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/rich-text": "file:../rich-text",
 		"lodash": "^4.17.11",
-		"memize": "^1.0.5",
```

Comment:
`package-lock.json` should have been updated as a result of this change. There's local revisions after running `npm install` in master.

Aside: Travis should be catching this, and it's unclear why it's not.

Reply:
See #14808